assets/src/components/detours/activeDetourPanel.tsx

```diff
@@ -85,8 +91,13 @@ export const ActiveDetourPanel = ({
       </Panel.Body.ScrollArea>
 
       <Panel.Body.Footer>
-        <Button className="flex-grow-1 m-3" onClick={onDeactivateDetour}>
-          Deactivate Detour
+        <Button
+          variant="ui-alert"
+          className="flex-grow-1 m-3 icon-button"
```

Comment:
can you try `icon-link` instead?
We shouldn't need custom css for this AFAIK
```suggestion
          className="flex-grow-1 m-3 icon-link"
```

Reply:
Oh that does get us most of the way there! Cool. Now need `text-light`, and I can do that inline.

One other thing though: the gap for `icon-link` is .375rem, while our [design system](https://www.figma.com/design/ui7Ni9uoeRrX7NLp3Hh5a6/Skate-Component-Library-(Bootstrap-5)?node-id=2658-914&t=UXiDIp8EJRIjVaNT-4) uses .5rem. It seems we should make an override for the gap of icon-link?

Reply:
I'd go back to Design and ask if this is something we can standardize on. Ideally either they adopt the Bootstrap default, or we change the `$icon-link-gap` variable.

Reply:
I chitchatted with Brian, and realized that it pretty much is standardized. All instances where we use `icon-link` as a utility in our code actually has the wrong margin, compared with the designs. The designs, whenever there is an icon with text, have a .5rem padding between the icon and text. I'm [checking](https://mbta.slack.com/archives/C024N3SKX33/p1726075371554479) with the designers now to see if this will continue to be a standard going forward

Reply:
Got confirmation from both Kathleen and Brian that .5rem is the standard gap going forward